k8s/operators/src/diskpool/crd.rs

```diff
@@ -66,11 +69,26 @@ pub enum PoolState {
     Error,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, JsonSchema)]
+#[non_exhaustive]
+pub enum PoolStatus {
+    /// State is Unknown
```

Comment:
doc comments should be `C`apitalized and end with a `.`

Reply:
Done